src/sql/src/plan/statement/show.rs

```diff
@@ -518,7 +518,8 @@ pub fn show_indexes<'a>(
             idx_cols.index_position AS seq_in_index,
             obj_cols.name AS column_name,
             idx_cols.on_expression AS expression,
-            idx_cols.nullable AS nullable
+            idx_cols.nullable AS nullable,
+            idxs.enabled AS enabled
```

Comment:
I don't think this is the same (presuming "same" regards [this comment](https://github.com/MaterializeInc/materialize/pull/7887/files#r690703928)) because this is the output of `SHOW INDEXES`, unless you're suggesting we don't want to express index's availability to users?

Reply:
Same answer as my above comment: as long as users have some way of discovering things that would materialize, then whatever is fine here.